supplementary_style_guide/glossary_terms_conventions/general_conventions/p.adoc

```diff
@@ -381,6 +381,19 @@ Use "plugin" rather than "plug-in", unless you are updating existing content tha
 
 *See also*:
 
+[[previously]]
+==== image:images/caution.png[caution] Previously (adverb)
+*Description*: In most content, you can use "previously" to refer to something that was true or in effect at an earlier time but is no longer true or in effect now.
+
+However, When drafting release notes, refrain from using "previously" to describe earlier releases. Instead, use "before this release," as shown in this guide's xref:release-notes[Release notes] section. This is because, in written release notes that might span multiple releases, the temporal context of "previously" can become ambiguous and cause confusion.
```

Comment:
```suggestion
However, when drafting release notes, refrain from using "previously" to describe earlier releases. Instead, use "before this release," as shown in this guide's xref:release-notes[Release notes] section. This is because, in written release notes that might span multiple releases, the temporal context of "previously" can become ambiguous and cause confusion.
```